R/zoop_downloader.R

```diff
@@ -610,14 +756,15 @@ Zoopdownloader <- function(
                     SizeClass = "Macro",
                     CPUE = .data$AdjCount/.data$volume, #add variable for data source and calculate CPUE
                     Microcystis = dplyr::recode(.data$Microcystis, `1=absent`="1", `2=low`="2"))%>%
-      dplyr::select(.data$Source, .data$Date, .data$Datetime,
+      dplyr::select(.data$Source, .data$Date, .data$Datetime, .data$Latitude, .data$Longitude,
```

Comment:
As before, wondering about the difference between these coordinates and those stored in our stations file

Reply:
Yeah, they are going to be a little different. I'm happy to do left_join stations instead, if you think it's better to be consistant with the other datasets. I feel like the exact locaiton of the trawl is better, personally.